clients/client/src/main/java/org/projectnessie/client/api/NessieApiV1.java

```diff
@@ -123,4 +123,16 @@
    * <p>See {@code org.projectnessie.client.StreamingUtil} in {@code nessie-client}.
    */
   GetRefLogBuilder getRefLog();
+
+  /** Fetch a single {@link org.projectnessie.model.Namespace}. */
+  GetNamespaceBuilder getNamespace();
+
+  /** Fetch one or more {@link org.projectnessie.model.Namespace}s based on a given prefix. */
+  GetNamespacesBuilder getNamespaces();
```

Comment:
Same naming concern: `GetNamespaceBuilder` is _very_ similar to `GetNamespacesBuilder`... could the latter be renamed?

Reply:
renamed to `getMultipleNamespaces`